src/Engines/Extensions/engine.rawTexture.ts

```diff
@@ -171,6 +171,42 @@ declare module "../../Engines/engine" {
          * @param textureType defines the texture Type (Engine.TEXTURETYPE_UNSIGNED_INT, Engine.TEXTURETYPE_FLOAT...)
          */
         updateRawTexture3D(texture: InternalTexture, data: Nullable<ArrayBufferView>, format: number, invertY: boolean, compression: Nullable<string>, textureType: number): void;
+

```

Comment:
I guess this part and the implementation could be moved to a dedicated engine.rawTextureArray file to help with tree shaking ?

Reply:
OK done